.github/workflows/build-wasm.yml

```diff
@@ -0,0 +1,117 @@
+jobs:
+  build-runtime-wasms:
+    # environment: ${{ github.ref == 'refs/heads/main' && 'production' || '' }}
+    # environment: production
+    # Benchmark (with cache hit)
+    # ubuntu-latest 50 min
+    # ubuntu-latest-4-cores 34 min
+    runs-on:  ubuntu-latest-4-cores
+    strategy:
+      matrix:
+        # To test until I get one right
+        # target: [build-runtime, build-runtime-fast, build-runtime-testnet]
+        # package: [centrifuge-runtime, altair-runtime]
+        target: [build-runtime, build-runtime-fast]
+        package: [altair-runtime, centrifuge-runtime]
+        # rust_version: ["1.69.0"]
+    name: ${{ matrix.target }}-${{ matrix.package }}
+    # Cannot make sccache work with the srtool container
+    # env:
+      # SCCACHE_GHA_ENABLED: "true"
+      # RUSTC_WRAPPER: "sccache"
+    steps:
+    # PREP  
+      - name: Check out code
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab #3.5.2
+
+      - name: Prep build on Ubuntu
+        uses: ./.github/actions/prep-ubuntu
+        # Use default toolchain
+        # with:
+        #   RUST_TOOLCHAIN: ${{ matrix.rust_version }}
+
+      # sccache does not work inside the srtool container
+      - uses: Swatinem/rust-cache@e207df5d269b42b69c8bc5101da26f7d31feddb4 # 2.6.2
+        with:
+          prefix-key: build #-${{ matrix.package }}
+          shared-key: wasm
+          workspaces: |
+            .
+            ./target/srtool
+          cache-directories: ./runtime
+          # save-if: ${{ github.ref == 'refs/heads/master' }}
+    # BUILD
+      - 
+        id: buildopt
+        run: |
+          if ${{ matrix.target == 'build-runtime-fast'}} ; then
+            echo "BUILD_OPTS=--features=fast-runtime" >> GITHUB_ENV
+          fi
+          echo "BUILD_OPTS=${{ env.BUILD_OPTS }}"
+
+      - name: Run srtool
+        run: >
+          docker run --rm  --user root --platform=linux/amd64
+          -e PACKAGE=${{ matrix.package }} -v /home/runner/.cargo:/cargo-home  
+          -v ${{ github.workspace }}:/build paritytech/srtool:1.73.0-0.12.0
+          /srtool/build 
+
+      # - name: Run Docker SRTool
+      #   uses: addnab/docker-run-action@4f65fabd2431ebc8d299f8e5a018d79a769ae185 #v3
+      #   env:
+      #     RUSTC_VERSION: ${{ matrix.rust_version }}
+      #     TARPAULIN_VERSION: 0.19.1 # travis-ci codecov generator
+      #   with:
+      #     image: paritytech/srtool:${{ matrix.rust_version }}
+      #     options: --user root -v /home/runner/.cargo:/cargo-home -v ${{ github.workspace }}:/build -e BUILD_OPTS=${{ env.BUILD_OPTS || ''}} -e PACKAGE=${{ matrix.package }}
+      #     run: |
+      #       echo "---- Env VARS ----"
+      #       echo "BUILD_OPT=${BUILD_OPT}"
+      #       echo "PACKAGE=${PACKAGE}"
+      #       echo "---- List Cache Folder ----"
+      #       ls -la /cargo-home/
+      #       du -sh /cargo-home/*
+      #       echo "---- RUNNING BUILD ----"
+      #       /srtool/build --save
```

Comment:
Can be removed?

Reply:
I'm going to leave it here because it would help anyone trying to debug inside the srtool container as it allows for an "arbitrary" script execution rather than running /srtool/build only

Reply:
I would add a comment indicating just that